web/src/main/java/org/fao/geonet/kernel/search/classifier/ClassificationScheme.java

```diff
@@ -0,0 +1,129 @@
+//=============================================================================
```

Comment:
Followed this up with @pblain who wlll check the requirement here.  GeoNetwork have specific requirements re licensing for contributed code.